download/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Decred - downloads</title>
+  <!--<base href="https://www.decred.org">-->
+  <meta content="Decred - downloads" name="description">
+  <meta content="Decred - downloads" property="og:title">
+  <meta content="Decred - downloads" property="og:description">
+  <meta content="http://decred.org/content/images/ogMainSite.jpg" property="og:image">
+  <meta content="summary" name="twitter:card">
+  <meta content="width=device-width, initial-scale=1" name="viewport">
+  <link href="../content/css/normalize.css" rel="stylesheet" type="text/css">
+  <link href="../content/css/components.css" rel="stylesheet" type="text/css">
+  <link href="../content/css/decred.css" rel="stylesheet" type="text/css">
+  <script src="../content/js/modernizr.js" type="text/javascript"></script>
+  <!--  Custom favicon  -->
+  <link rel="apple-touch-icon" sizes="180x180" href="../content/images/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" href="../content/images/favicon/favicon-32x32.png" sizes="32x32">
+  <link rel="icon" type="image/png" href="../content/images/favicon/favicon-16x16.png" sizes="16x16">
+  <link rel="manifest" href="../content/images/favicon/manifest.json">
+  <link rel="mask-icon" href="../content/images/favicon/safari-pinned-tab.svg">
+  <link rel="shortcut icon" href="../content/images/favicon/favicon.ico">
+  <meta name="apple-mobile-web-app-title" content="deCRED - decentralized credit">
+  <meta name="application-name" content="deCRED - decentralized credit">
+  <meta name="msapplication-TileColor" content="#2973ff">
+  <meta name="msapplication-TileImage" content="./content/images/favicon/mstile-144x144.png">
+  <meta name="msapplication-config" content="./content/images/favicon/browserconfig.xml">
+  <meta name="theme-color" content="#ffffff">
+</head>
+<body class="download-page">
+  <div class="navigation w-clearfix">
+    <div class="navigation-button">
+      <div class="nav-open-close"></div>
+    </div>
+    <div class="link-section">
+      <div class="nav-links-wrap">
+        <a class="navigation-link" href="https://decred.org/#constitution">About</a>
+        <a class="navigation-link" href="https://decred.org/#guide">Getting Started</a>
+        <a class="navigation-link" href="https://decred.org/download/">Downloads</a>
+        <a class="navigation-link" href="https://docs.decred.org">Documentation</a>
+        <a class="navigation-link" href="https://blog.decred.org">Blog</a>
+        <a class="navigation-link" href="https://decred.org/#technology">Technical Brief</a>
+        <a class="navigation-link" href="https://stats.decred.org">Network Stats</a>
+        <a class="navigation-link" href="https://voting.decred.org">Voting</a>
+        <a class="navigation-link" href="https://decred.org/#modalOpen">Stakepools</a>
+        <a class="navigation-link" href="https://decred.org/brand/" target="_blank">Logo and Visuals</a>
```

Comment:
relative links added, the stakepools link needs to stay absolute to trigger the modalOpen